modules/manage/pages/iceberg/iceberg-topics-aws-glue.adoc

```diff
@@ -78,14 +86,24 @@ For more information on configuring IAM permissions, see the https://docs.aws.am
 
 == Configure authentication and credentials
 
-You can configure credentials for the AWS Glue Data Catalog integration in either of the following ways:
+ifndef::env-cloud[]
+You must configure credentials for the AWS Glue Data Catalog integration in either of the following ways:
 
 * Allow Redpanda to use the same `cloud_storage_*` credential properties configured for S3. If you do not configure the overrides listed below, Redpanda uses the same credentials for both S3 and AWS Glue. This is the recommended approach.
 * If you want to configure authentication to AWS Glue separately from authentication to S3, there are equivalent credential configuration properties named `iceberg_rest_catalog_aws_*` that override the object storage credentials. These properties only apply to REST catalog authentication, and never to S3 authentication:
 ** config_ref:iceberg_rest_catalog_aws_access_key,true,properties/cluster-properties[`iceberg_rest_catalog_aws_access_key`] overrides config_ref:cloud_storage_access_key,true,properties/cluster-properties[`cloud_storage_access_key`]
 ** config_ref:iceberg_rest_catalog_aws_secret_key,true,properties/cluster-properties[`iceberg_rest_catalog_aws_secret_key`] overrides config_ref:cloud_storage_secret_key,true,properties/cluster-properties[`cloud_storage_secret_key`]
 ** config_ref:iceberg_rest_catalog_aws_region,true,properties/cluster-properties[`iceberg_rest_catalog_aws_region`] overrides config_ref:cloud_storage_region,true,properties/cluster-properties[`cloud_storage_region`]
 ** config_ref:iceberg_rest_catalog_aws_credentials_source,true,properties/cluster-properties[`iceberg_rest_catalog_aws_credentials_source`] overrides config_ref:cloud_storage_credentials_source,true,properties/cluster-properties[`cloud_storage_credentials_source`]
+endif::[]
+
+ifdef::env-cloud[]
+You must configure credentials for the AWS Glue Data Catalog integration using the following properties:
```

Comment:
Just double checking, are these always needed? Do the cloud_storage_* credentials not work in cloud?

Reply:
That's right -- `cloud_storage_*` properties are not exposed in Cloud because we manage the cloud storage bucket.